modules/controller/index.js

```diff
@@ -60,7 +61,7 @@ Controller.prototype.create = function (opts) {
       var payload = createResponse(err, data, _.extend({}, opts, {
         type: type
       }));
-      responder(payload, request, response);
+      this.responder(payload, request, response);
```

Comment:
the context of `this` here will be wrong, maybe to `var responder = this.responder` outside the closure like the rest?